packages/labs/virtualizer/package.json

```diff
@@ -50,7 +50,7 @@
       ]
     },
     "build:copy-polyfill": {
-      "command": "mkdir -p polyfills/resize-observer-polyfill && cp src/polyfills/resize-observer-polyfill/ResizeObserver.js polyfills/resize-observer-polyfill/ResizeObserver.js",
+      "command": "node -e \"require('fs').mkdirSync('polyfills/resize-observer-polyfill', {recursive: true})\" && cp \"src/polyfills/resize-observer-polyfill/ResizeObserver.js\" \"polyfills/resize-observer-polyfill/ResizeObserver.js\"",
```

Comment:
I don't know how much effort we really want to put in for Windows support but `cp` is not available in Windows when using `cmd.exe`. It is available in PowerShell, and Git Bash though.

It would be more compatible to write a Node script that does it all together.

Probably way extra to bring in additional dependencies for this but there are things like https://www.npmjs.com/package/shx which allows using shell commands inline.

Reply:
Ah, I see you've noted this could be part of rollup. I agree that would be a better long term solution. :)

Reply:
Great point about `cp`. I was using Git Bash on Windows so didn't get impacted by the missing `cp`.

Downgrading this PR back to a draft until this is a more complete solution.

Reply:
2024, work fine. good.